Xwt.Gtk.Mac/GtkMacAccessibleBackend.AccessibilityHelper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using Xwt.GtkBackend;
+
+namespace Xwt.Gtk.Mac
+{
+	public partial class GtkMacAccessibleBackend : AccessibleBackend
+	{
+		static Xwt.Mac.AccessibleBackend xamMacA11yHelper;
+		static GtkMacAccessibleBackend ()
+		{
+			xamMacA11yHelper = new Xwt.Mac.AccessibleBackend ();
```

Comment:
I'd rather move the static part of Xwt.XamMac/Xwt.Mac/AccessibleBackend.AccessibilityHelper.cs to a separate file, linked in this project and then copy the instance code here.